source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet

```diff
@@ -55,25 +56,31 @@ local question(title) = {
   ],
 };
 
-local nonProxyTitle = 'In the last seven days, were you doing any of the following?';
-local proxyTitle = {
-  text: 'In the last seven days, was <em>{person_name}</em> doing any of the following?',
-  placeholders: [
-    placeholders.personName(),
-  ],
-};
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'In the last seven days, was <em>{person_name}</em> doing any of the following?',
+    placeholders: [
+      placeholders.personName(),
+    ],
+  }
+  else 'In the last seven days, were you doing any of the following?'
+);
+
+local questionDescription(isProxy) =
+  if isProxy then 'If they have a job but have been off work on <em>furlough</em>, select “Temporarily away from work ill, on holiday or temporarily laid off”'
+  else 'If you have a job but have been off work on <em>furlough</em>, select “Temporarily away from work ill, on holiday or temporarily laid off”';
 
 {
   type: 'Question',
   id: 'employment-status-last-seven-days',
   page_title: 'Employment status in the last seven days',
   question_variants: [
     {
-      question: question(nonProxyTitle),
+      question: question(questionTitle(false), questionDescription(false)),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle),
+      question: question(questionTitle(true), questionDescription(true)),
```

Comment:
An `isProxy` variable should be passed to `question`, and the `questionTitle` and `questionDescription` methods called from there. See https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet for an example.